src/app/data-planes/views/DataPlaneTrayView.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <RouteView name="data-plane-tray-view">
+    <AppView>
+      <template #title>
+        <div class="tray-title-wrapper">
+          <img
+            aria-hidden="true"
+            src="@/assets/images/icon-wifi-tethering.svg?url"
+          >
+
+          <h2 class="tray-title">
+            <RouterLink
+              :to="{
+                name: 'data-plane-detail-view',
+                params: {
+                  dataPlane: props.data.name,
+                },
+              }"
+            >
+              <RouteTitle
+                :title="t('data-planes.routes.item.title', { name: props.data.name })"
+                :render="true"
+              />
+            </RouterLink>
+          </h2>
+        </div>
+      </template>
+
+      <div class="stack">
+        <div>
+          <h3>{{ t('data-planes.routes.item.overview') }}</h3>
+
+          <div class="mt-4 stack">
+            <DataPlaneDetails :dataplane-overview="props.data" />
+
+            <DataPlaneDateDetails :dataplane-overview="props.data" />
+          </div>
+        </div>
+
+        <div>
```

Comment:
I think we should include the YAML here. I'd also would like that in 2.5 we don't use any more data in the tray views than the one we already have from the list view.

We can always make it better later and I'd rather keeping it simple with our time constraints.

Reply:
What data should use here then? Exactly what we have in the list view or a subset (I can make an educated guess in that case).

Also, are you fine with _moving_ both "Last updated" and "Certificate info" into the summary and out of the table?

Reply:
FWIW, I think we added the certificate info into the rows so it was easy to see multiple at a glance. Moving it out means we can't do that, you'd have to click on each row to see it.

Reply:
Sorry bad wording. What I mean "what we have in the list view" is "what data is returned by the API from a list view". We can show more info that what is in each rows.

Reply:
Oh, I see. I have definitely misunderstood that then.

In that case I’d like consider the following things to show here (if there is enough space I’d like to show all):

- Status (in list as well)
- Last updated (no longer in list, I don’t think it’s important enough)
- Certificate info (in list as well)
- Tags (not in list, only detail)
- Versions for envoy, kumaDp, etc. (not in list, only detail)

Question: Should we show the YAML? That’s not in the list view API data, so by your rule that’d be a No

Reply:
Let's not show the YAML

Reply:
I'm happy with the rest of this info

Reply:
I removed the YAML now. The tray views now show the data as outlined above.